WtWSMS/localization/english/bookmark/bm_476_l_english.yml

```diff
@@ -2,84 +2,84 @@
  
- bookmark_476_julius_nepos:0 "Julius"
- bookmark_476_julius_nepos_desc:0 "Julius Nepos, seen by some as the last legitimate Western Roman Emperor, was its de facto ruler following the ascension of Romulus Augustus, and then the barbarian Odovacar. In control of a mere fragment of his empire, a small holdout in Dalmatia, Julius gears up for a massive invasion to reclaim his rightful land - starting with the Italian Peninsula.\n\n#bold Will the true claimant succeed?#!"
+ bookmark_476_julius_nepos: "Julius"
+ bookmark_476_julius_nepos_desc: "Julius Nepos, seen by the #italic Pars Orientalis#! as the last legitimate Western Roman Emperor, rules a small strip of land along the Dalmatian coast, following his ousting by the cunning Orestes.\n\nIn control of a small holdout, Julius gears up for an invasion to reclaim his rightful land - starting with the Italian Peninsula.\n\n#bold Will the true claimant succeed, or will the dreams of the #italic Pars Occidentalis#! die with him?#!"
  
- bookmark_476_odoacer:0 "Odovacar"
- bookmark_476_odoacer_desc:0 "The mighty Odovacar was a citizen and general of Rome, and member of a Germanic #italic foederati#!. Following a period of rebellion and usurpation, Odovacar led the Germanic #italic foederati#! stationed in Italy against Romulus Augustus, displacing the boy emperor and founding the Kingdom of Italy. Thoroughly an integrated Roman citizen, his efforts were met with praise and reward from Zeno, the Emperor-in-the-East, as he was allowed to govern Italy as an independent Roman province. However, left without the support of the Eastern Roman Empire, Odovacar must meet the threats that for so long had destabilized the Western Roman Empire."
+ bookmark_476_odoacer: "Odovacar"
+ bookmark_476_odoacer_desc: "Odovacar is a citizen and general of Rome, and member of a Germanic #italic foederati#!. When the #foederati#! were denied lands in Italy, Odovacar led them against Romulus Augustus, displacing the boy emperor and founding the Kingdom of Italy. Thoroughly an integrated Roman citizen, his efforts were met with praise and reward from Zeno, the Emperor-in-the-East, as he was allowed to govern Italy as an independent Roman province. However, left without the support of the Eastern Roman Empire, Odovacar must meet the threats that for so long had destabilized the Western Roman Empire.\n\n#bold Can Odovacar secure a place for his people, or will he meet the same end that befell other Roman pretenders?#!"
```

Comment:
Maybe not Roman pretender but rather Germanic conqueror?

Reply:
As a Germanic conqueror, he arguably failed where others (like Theodoric, Geiseric, and Alaric) succeeded (given his Italian Kingdom didn't even last twenty years, so I'd go more like:

> Can Odovacar secure a place for his people, like other Germanic conquerors, or will he fail where Alaric and Geiseric succeeded?